client/tests/e2e/specs/test_users_page.ts

```diff
@@ -20,35 +20,98 @@ describe('Check active users page', () => {
     cy.get('#activate-users-ms-action-bar').find('#list-view').should('have.attr', 'disabled');
     cy.get('#activate-users-ms-action-bar').find('#button-invite-user').contains('Invite a user');
     cy.get('.user-list-header').find('ion-checkbox').should('not.have.class', 'checkbox-checked');
-    cy.get('.users-container').find('.user-list-item').as('userItems').should('have.length', 4);
+    cy.get('.users-container').find('.user-list-item').as('userItems').should('have.length', 13);
```

Comment:
You're going way too far with the tests. Having 3-4 users should be enough, there's no need to have 12 of them that are mostly similar and checking that each invidual checkbox is checked or not. These tests are already a chore to maintain, having a lot of copy/paste makes it even more of a struggle.

Just keep the sort test, check that the first and last of the list are what we expect, and it's more than enough.